shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLDALStatementSQLVisitor.java

```diff
@@ -452,6 +459,44 @@ public ASTNode visitShowCreateTrigger(final ShowCreateTriggerContext ctx) {
         result.setName(((IdentifierValue) visit(ctx.triggerName())).getValue());
         return result;
     }
+
+    @Override
+    public ASTNode visitShowRelaylogEvent(final MySQLStatementParser.ShowRelaylogEventContext ctx) {
+        MySQLShowRelaylogEventsStatement result = new MySQLShowRelaylogEventsStatement();
+        ChannelNameSegment channel = (ChannelNameSegment) visit(ctx.channelName());
+        result.setChannel(channel);
+        Optional.ofNullable(ctx.logName())
```

Comment:
@flycash Can you use the new visit method to encapsulate this logic? The use of lambda expressions here does not seem very clear.